benchexec/tablegenerator/columns.py

```diff
@@ -162,7 +162,7 @@ def format_value(self, value, isToAlign=False, format_target="html"):
         if format_target not in POSSIBLE_FORMAT_TARGETS:
             raise ValueError('Unknown format target')
 
-        if value is None:
+        if value is None or value == '':
```

Comment:
This should be `if value:`

Reply:
But we have to do proper formatting if `value == 0`, and `not value` would also be true for `value == 0`.